### PR TITLE
Alter existing "deprecated" function to return all migration notices 

### PR DESCRIPTION
**Pull request by rajansandeep**
Alter existing "deprecated" function to return all migration notices.
Also removes the "removed" fn since it's now merged with the "deprecated" fn

### CODE REVIEW

File: kubernetes/corefile-tool/README.md [hunk 1]
```diff
@@ -52,6 +55,11 @@ corefile-tool default --k8sversion 1.4.0 --corefile /path/to/Corefile
 corefile-tool deprecated --from 1.4.0 --to 1.5.0 --corefile /path/to/Corefile
 ```
 
+```bash
+# See ignored plugins CoreDNS from v1.4.0 to v1.5.0. 
+corefile-tool deprecated --from 1.4.0 --to 1.5.0 --corefile /path/to/Corefile
```
Comment (chrisohaver) [inline kubernetes/corefile-tool/README.md]:
```suggestion
corefile-tool ignored --from 1.4.0 --to 1.5.0 --corefile /path/to/Corefile
```

**Comment by rajansandeep** [comment]
>I'm thinking it may make more sense to merge the three funcitons (Deprecated, Ignored, and Removed) into one function (Deprecated) that just returns a list of all deprecations/no-ops/removals ... perhaps even adds (though that may look strange). Otherwise a user would have to call each one and merge the results. What do you think?

@chrisohaver I kept all the existing function but added a new one which gets cumulative of all the fns.

**Comment by chrisohaver** [comment]
I think we should remove the "ignored" and "removed" functions, and alter the existing "deprecated" function to return all migration notices ("deprecated", "ignored", "removed", "newdefault").

I think this makes more sense from a user perspective.  From a user perspective, removal of a feature, or a feature becoming a no-op is a kind of deprecation.

I don't think the individual functions have value on their own.

**Pull request merged**